caster-front/src/components/GraphPlayer.vue

```diff
@@ -267,14 +292,40 @@ const convertButtonType = (b: ButtonType): ElButtonType => {
       return ElButtonType.Default;
   }
 };
+
+const sendStreamVariableMutation = useSendStreamVariableMutation();
+
+// template functions
+const startStream = async () => {
+  // TODO: Do we still need this?
+  // I don't think so, since the page should show an error if it can't get a stream
+  // if (data.value?.streamInfo.__typename === "NoStreamAvailable") {
+  //   ElMessage.error("Can not start an unassigned stream");
+  //   return;
+  // }
```

Comment:
I sometimes added these kind of things in order to access the non `NoStreamAvailable` data of our subscription - I think it is safe to remove it :)